### PR TITLE
[hotfix]Fix the template for backupInfrastructure deletionGracePeriodHoursByPurpose in gardener chart

### PR DESCRIPTION
**Pull request by swapnilgm**
Signed-off-by: Swapnil Mhamane <swapnil.mhamane@sap.com>

**What this PR does / why we need it**:
This PR fixes the template for backupInfrastructure deletionGracePeriodHoursByPurpose in gardener chart.
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
Please checkout the Garedner chart rendering using `helm template`
**Release note**:

```improvement operator
Fixed the template for backupInfrastructure deletionGracePeriodHoursByPurpose in gardener chart
```

### CODE REVIEW

**Comment by swapnilgm** [comment]
cc: @shreyas-s-rao

File: charts/gardener/charts/runtime/templates/configmap-controller-manager.yaml [hunk 1]
```diff
@@ -104,7 +104,10 @@ data:
         deletionGracePeriodHours: {{ .Values.global.controller.config.controllers.backupInfrastructure.deletionGracePeriodHours }}
         {{- end }}
         {{- if .Values.global.controller.config.controllers.backupInfrastructure.deletionGracePeriodHoursByPurpose }}
-        deletionGracePeriodHoursByPurpose: {{ .Values.global.controller.config.controllers.backupInfrastructure.deletionGracePeriodHoursByPurpose }}
+        deletionGracePeriodHoursByPurpose:
+          {{- range $key, $value := .Values.global.controller.config.controllers.backupInfrastructure.deletionGracePeriodHoursByPurpose }}
```
Comment (shreyas-s-rao) [inline charts/gardener/charts/runtime/templates/configmap-controller-manager.yaml]: Why simply not use 

```{{ toYaml .Values.global.controller.config.controllers.backupInfrastructure.deletionGracePeriodHoursByPurpose | indent 10 }}```

**Review by shreyas-s-rao — COMMENTED** [review]
Thanks for raising the fix PR. Just a suggestion to simplify populating the value for `deletionGracePeriodHoursByPurpose `

**Comment by swapnilgm** [comment]
@shreyas-s-rao Done, PTAL.

**Review by shreyas-s-rao — APPROVED** [review]
LGTM

**Review by zanetworker — APPROVED** [review]

**Pull request merged**